app/views/layouts/application.html.erb

```diff
@@ -7,15 +7,13 @@
     <%= stylesheet_link_tag "application", media: "all", 'data-turbolinks-track': "reload" %>
     <%= stylesheet_link_tag "application", "http://fonts.googleapis.com/css?family=Roboto" %>
     <%= javascript_include_tag "application", 'data-turbolinks-track': "reload" %>
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
   </head>
 
-  <body>
+  <body class="<%= "hands-bg" if login_page? %>">
```

Comment:
do we want this on the body?

Reply:
yeah we probably don't want this in body